typing/typedtreeIter.ml

```diff
@@ -249,7 +250,7 @@ module MakeIterator(Iter : IteratorArgument) : sig
         exp.exp_extra;
       begin
         match exp.exp_desc with
-          Texp_ident (path, _, _) -> ()
+        | Texp_ident (path, _, _) -> ()
```

Comment:
There are many occurrences of '|' on the first branch in the compiler code base. I don't think anyone would complain about this style.